### PR TITLE
fix #5942 match silas notebook

### LINKED ISSUE #5942: matching notebook vs sirepo silas output
This simulation:
https://alpha.sirepo.com/silas#/beamline/Hu8T0sP4

should match the output from this notebook:
https://github.com/radiasoft/rslaser_scratch/blob/main/notebooks/propagate/single_crystal_pass_20230606.ipynb

### PR DESCRIPTION
**Pull request by moellep**
- fix problem with axis ticks for range [-0.02, 0]
- fix broken silas percentComplete

### CODE REVIEW

File: sirepo/template/silas.py [hunk 1]
```diff
@@ -425,53 +436,49 @@ def _z_label(plot_type):
...
-        return PKDict(
-            title=_title(plot_type, slice_index),
-            x_range=[r.x[0], r.x[1], len(z)],
-            y_range=[r.y[0], r.y[1], len(z[0])],
-            x_label="Horizontal Position [m]",
-            y_label="Vertical Position [m]",
-            z_label=_z_label(plot_type),
-            z_matrix=z,
-        )
+    tries = 3
```
Comment (gurhar1133) [inline sirepo/template/silas.py:439]: doesn't seem like needs `tries` var

**Review by gurhar1133 — CHANGES_REQUESTED** [review]
There seems to be a bug where crystal plots arent showing on the beamline tab.

File: sirepo/template/silas.py (hunk 1 above)
Comment (gurhar1133) [reply sirepo/template/silas.py:439]: But maybe could be constant

Comment (moellep) [reply sirepo/template/silas.py]: I moved it to the top of the file as _MAX_H5_READ_TRIES = 3

**Comment by moellep** [comment]
@gurhar1133 I think I've fixed those 2 issues - please give it another look

**Review by gurhar1133 — APPROVED** [review]
Yeah, I see the crystal plots on beamline tab now

**Pull request merged**